codecov_cli/helpers/request.py

```diff
@@ -157,3 +160,27 @@ def log_warnings_and_errors_if_any(
         logger.error(f"{process_desc} failed: {sending_result.error.description}")
         if fail_on_error:
             exit(1)
+
+
+def _sanitize_request_result(result: RequestResult):
+    if not hasattr(result, "text"):
+        return result
+
+    try:
+        text_as_dict = json.loads(result.text)
+        token = text_as_dict.get("repository").get("yaml").get("codecov").get("token")
+        if token:
+            sanitized_token = str(token)[:1] + 18 * "*"
```

Comment:
why include the first character?

Reply:
@matt-codecov we do this in other parts of the system. the intention is that it helps to diagnose issues (e.g. copying `CODECOV_TOKEN` as the token)